### DISCUSSION TITLE
[bug] Identies abuse - a security issue

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#14936] [Feature improvement]
Fact that users can use identities of other users is a critical security breach.
This is long standing design error intentionally overseen or ignored.
It would be enough to make little projection of 200 users working in the same app. 
I never been on it unless I see latest discussions and reports. It is nothing developers can help (please don't speak about money) as design goes too deep and hooking decision of using Roundcube as an email front-end. Certainly in out-of-the-box configuration YF cannot be used by larger distributed team of users

**Comment by diderich** [comment]
Please explain, referencing the actual fuction(s) that do no work as intended, e.g., through a use case. I don't understand the secuity problem you try to describe.

**Comment by waran70** [comment]
The problem what had been raised is that Roundcube allowing every user to send email with identity of other users. Just try. I did. 
look here: 
![image](https://user-images.githubusercontent.com/6852348/125085221-869cd800-e0f4-11eb-8b7b-c4ed15c99e07.png)

Even you configure system to use one identity, user still have option to change signature. Yes, just imagine as bug reporting guy said: 200 users, and each in different city or country. 
Email system can be different, you can have own server, or use gmail for instance. Gmail allows for many identities in one email account, it is cool, but not completely. Identity means access. CRM is a multi-access system and each user should have independent identity but admin should not be forced to create separate email for each user.

**Reply by s2b-git** [reply]
Yes, users can change identity in Roundcube. At least in Yetiforce 6.1. But if you configure workflows, you can use only tickets for Email communication - it just longer way

**Reply by mariuszkrzaczkowski** [reply]
Creating a good e-mail client comparable to gmail, roundcube, etc. is quite a cost.  I am not saying not only who will do it, we have already thought about it several times.
The enormous amount of work discouraged us from doing

**Reply by waran70** [reply]
Good one not necessary. Send, CC, BCC, attach file, attach template. Quitting roundube will get users impression of more integrated system rather than one relaying on third-party-looking-very-much extension. Mail scanner is where email power of yetiforce is. Getting back to vtiger basic thing would be enough. As it were said - if users want robust mail client, go enjoy outlook or tunderbird. Custom integrations is where some coins may wait for developer.

**Reply by mariuszkrzaczkowski** [reply]
vtoger's mail client had security flaws

**Comment by IanMuller** [comment]
Au, it has been moved from Issues to Discussion...? So using identity of another user isn't any issue? How it come?

**Reply by mariuszkrzaczkowski** [reply]
It's not an application error, but a request for something new
